tools/changelog.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""Grab the changelog for issues."""
+
+import argparse
+import datetime
+import os
+import sys
+from urllib import quote_plus
+from urlparse import urlunsplit
+
+import requests
+
+# This will start an initialization process.
+sys.path.append(os.path.realpath(os.getcwd()))
+from config.secrets import OAUTH_TOKEN
+
+# Config
+GITHUB_API = 'api.github.com'
+ROOT_REPO = '/repos/webcompat'
+REPO_TEST = 'webcompat-tests'
+REPO_BUGS = 'webcompat.com'
+QUERY_PARAM = 'labels'
+QUERY_VALUE = 'status: add to changelog'
+HEADERS = {'Accept': 'application/vnd.github.v3+json',
+           'Authorization': 'token {token}'.format(token=OAUTH_TOKEN),
+           'User-Agent': 'webcompat/webcompat-bot'}
+# Templates for producing the changelog
+LINE_TEMPLATE = '* {title} [Pull #{number}]({url})\n'
+LOG_TEMPLATE = """
+
+X.X.X - {date}
```

Comment:
Ooh, one more bit of feedback since I just used (a modified version of) this... Can we do `## X.X.X` instead of the plain string? That matches the markdown format in CHANGELOG.md. Thanks!

Reply:
done.